paradox-nakadi-consumer-partitioned-zk/src/main/java/de/zalando/paradox/nakadi/consumer/partitioned/zk/ZKConsumerSyncOffsetManagement.java

```diff
@@ -58,14 +67,19 @@ public void flush(final EventTypePartition eventTypePartition) {
     }
 
     @Override
-    public void error(final Throwable t, final EventTypePartition eventTypePartition) {
+    public void error(final Throwable t, final EventTypePartition eventTypePartition, @Nullable final String offset,
+            final String rawEvent) {
 
         // it will unsubscribe reactive receiver
         if (ThrowableUtils.isUnrecoverableException(t)) {
             LOGGER.error("Error [{}] reason [{}]", eventTypePartition, getMessage(t));
             ThrowableUtils.throwException(t);
         } else {
-            LOGGER.error("Error [{}] reason [{}]", eventTypePartition, getMessage(t), t);
+
+            eventErrorHandlers.forEach(eventErrorHandler ->
+                    eventErrorHandler.onError(t, eventTypePartition, offset, rawEvent));
+
+            LOGGER.error("Error [{}] reason [{}] raw event [{}] ", eventTypePartition, getMessage(t), t, rawEvent);
```

Comment:
[C] `t` should be the last argument. Also see http://slf4j.org/faq.html#paramException